build_tools/bundle.js

```diff
@@ -83,6 +83,18 @@ function aliasDirectories(mapDirectories, bundler) {
     });
 }
 
+function files(patterns) {
+    if (!Array.isArray(patterns)) {
+        return glob.sync(patterns);
+    }
+
+    var fileGroups = patterns.map(function(pattern) {
+        return glob.sync(pattern);
+    });
+
+    return Array.prototype.concat.apply([], fileGroups);
```

Comment:
yeah because glob.sync on 92 will return an array of files https://www.npmjs.com/package/glob#globsyncpattern-options. i'm flattening so that files function returns [file,file,file,file] rather than [[file],[file],[file],[file]]